src/base/net/smtp.cpp

```diff
@@ -163,16 +163,26 @@ void Smtp::sendMail(const QString &from, const QString &to, const QString &subje
     }
 
     // Connect to SMTP server
+    QStringList smtpServer;
+    quint16 smtpPort;
```

Comment:
Initialize variables when you declare them.
And add `const` whenever you can.

Reply:
Done.